index.js

```diff
@@ -54,27 +54,37 @@ function FirebaseApp(config) {
 export function FirebaseData(config = {}, initialState, ref = "") {
     const { database, app } = FirebaseApp(config)
     const data = Data(initialState)
+    const emptyArrayString = "$$firebaseEmptyArray"
```

Comment:
The only thing that could add some clarity is to name this const `firebaseEmptyArrayString` or `firebaseEmptyArrayStr`. But I also don't feel strongly about it. The code is pretty simple, so I think people can understand what's happening here.

Reply:
thats nice, done.